### DISCUSSION TITLE
"code" in configuration file

### DISCUSSION THREAD

**Discussion by actabi** [guerrerotook/securitas-direct-new-api#76] [Q&A] [answered]
Hello, I can't connect and I think it's because of the configuration file. We have to put the "code"... I don't know where to find it. What is it? Thanks for your help.

**Comment by SirLouen** [comment, marked as answer]
> Hello, I can't connect and I think it's because of the configuration file. We have to put the "code"... I don't know where to find it. What is it? Thanks for your help.

I'm also setting the configuration.yaml but I believe that is the 4-digit code that you can configure in your verisure user

**Comment by SirLouen** [comment]
Also be aware that if you are using a Verifast without number keyboard, I'm not 100% if this component could be configured.

**Comment by actabi** [comment]
I have phoned Verisus service and they gave me the 4 digit code. :-) Thank you for your help.

**Comment by SirLouen** [comment]
Can you put me a photo of your central unit to see if this is the same as mine?

I have multiple Securitas installations and have multiple different devices, one of them is this one that I cannot control with HA because I don't have this 4-digit, so I would like to check it also.

**Reply by actabi** [reply]
![20220913_123427](https://user-images.githubusercontent.com/4301546/189902854-839675a6-1dad-4300-b453-ba6c03e88298.jpg)

**Reply by SirLouen** [reply]
Have you been able to configure it with such code?

I've been talking with the technical support and they only have the disarm code not the arm code (and the coercion code which is the same as the master code)

So weird this old alarm model. 

The newer alarm is simpler.

**Comment by SirLouen** [comment]
Yep, that is the Verifast one of the two I have

Do you see the numbers around? Theoretically, you could input the number with them, and it should work (try it, arm the alarm, and then disarm it with the PIN they gave you). 

From what I saw there was a "master code" (6789) since they never touch this and noone uses it (this is a very insecure alarm, and they ended deprecating it because it was too easily sabotaged)

The procedure of this alarm is too straight forward: You can arm and disarm it with two buttons (lock + mode to disarm and mode to arm it). So basically if you had the central unit in a room without cams or sensors, and someone manage to go in there, they could disarm the whole house with ease. 

My recommendation is that you kindly ask them to change your alarm system ASAP. They are switching for 29€ + 2 years of renewal at least in Spain.

**Comment by Cannone2024** [comment]
Hallo everyone, I installed the securitas component but when I send the activation or deactivation commands I get the following error:
alarm_control_panel/alarm_disarm argument of type 'None Type' is not iterable.
Can you help me?
